docs/source/sharing.rst

```diff
@@ -0,0 +1,93 @@
+Sharing
```

Comment:
@polyester if there would be a PLIP and the framework team would signal their consent, I guess we could decide to rename the "sharing" endpoint to "access" in plone.api. Though, right now this seems to be a bit premature, what do you think?